### ISSUE TITLE
Confusion over MT6781…

### ISSUE THREAD

**Issue by navid-zamani** [bkerler/mtkclient#774]
Hello… you write that the MT6781 is not supported currently.  
But in bug #141, apparently the user was successful with the MT6781. But there is no information about how…  
So… is it possible?

I have a Blackview BV880 with such a chip, and I’m a programmer and very experienced in Linux. So I thought maybe I can help.

I cannot send you my phone, as I could barely afford it myself. ;)  
But if you tell me what you would do, I could do it (very precisely) and give you the output.  
Maybe even give you a shell on a system of mine, where you can interact with it, with me pressing the hardware buttons if required. :)  
As long as you don’t break it, of course, as I need to keep using it afterwards.

----

But one more thing: I tried looking into what doesn’t work myself, and did a stupid while at it: As my very first action, based on a bad guide, I ran:
```lang-bash
python3 mtk e metadata,userdata,md_udc
python3 mtk da seccfg unlock
python3 mtk reset
 ```
And of course all my data is gone (I have an `adb backup`. I just don’t know how to fully restore it yet. (Hey, it’s not a failure if I learned form it? Right? Right? XD)  
But the real problem is that I now get a `dm-verity` warning on boot, where it shuts down unless I press the power button.  
I tried `python3 mtk da seccfg lock`  but it did nothing. I presume it writes to places it should not write to, and corrupted things?  
Is my only path to undoing this, to flash the original firmware (which I don’t know how to find right now)? Or can something be done with your tool, to undo this warning?

**Comment by ZakitheBig** [comment]
I use the mtk gui 2.0 client which is an implementation of the mtk client but with more accessibility, I didn't find my official firmware so I go into brom and read all the partitions (except userdata) and then try something risky

Well, anyway, here is the firmware for your device:
https://naijarom.com/blackview-bv8800

**Comment by navid-zamani** [comment]
> I use the mtk gui 2.0 client which is an implementation of the mtk client but with more accessibility, I didn't find my official firmware so I go into brom and read all the partitions (except userdata) and then try something risky

I want to do that too. (But my vbmeta is empty now, also, mtk apparently client doesn’t support my chip, so I don’t trust it to reliably pull all partitions and put them back when I mess up and need to restore…, so …)
I managed to download boot_a (with SP Flash tool, which I got to almost work), patch it with Magisk, and write it back, but it did nothing. 

> Well, anyway, here is the firmware for your device: https://naijarom.com/blackview-bv8800
Those are the same files I found everywhere. They are for an older version, and doesn’t let me flash boot_b, which I suspect is required to get Magisk to work)

Man, what I nightmare.
